src/OpenDB/test/gcd_abstract_lef.lef.ok

```diff
@@ -0,0 +1,456 @@
+VERSION 5.8 ;
+BUSBITCHARS "[]" ;
+DIVIDERCHAR "/" ;
+MACRO gcd
+  FOREIGN gcd 0 0 ;
+  CLASS BLOCK ;
+  ORIGIN 0 0 ;
+  SIZE 200.26 201.6 ;
+  PIN clk
+    DIRECTION INPUT ;
+    USE SIGNAL ;
```

Comment:
This should be `USE CLOCK`

Reply:
Hmm, that's what OpenDB reports this BlockTerm as. Is there something in the def that denotes this as a clock net?

Reply:
Sta knows which nets are clock and we don't generally update the db with that info.  If this is important we could change that though it would be outside to scope of the writer itself.  That said I'm not clear it will make any real difference in having the the use set properly (I'm not aware of anything that cares).  @rovinski do you know of a use for USE?

Reply:
I couldn't tell you if it's going to cause problems - I'm just trying to adhere to the spec. It would be good to get this worked out eventually.